var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -203,6 +214,9 @@ def cmake_args(self):
                 define(project_name_base + "_ENABLE_CXX11", True),
                 define(project_name_base + "_ENABLE_Kokkos", False),
                 define(project_name_base + "_HIDE_DEPRECATED_CODE", False),
+                # Seacas MSVC tests are not tested with Zoltan
+                # which causes build errors, skip for now
+                define(project_name_base + "_ENABLE_Zoltan", not is_windows),
```

Comment:
(future PR) Interestingly, this package does not have an explicit `depends_on(zoltan)` - I assume `seacas` is building it itself on Linux?

Reply:
Right, `seacas` has its own `zoltan` on both Windows and Linux, `zoltan` in seacas is just broken on Windows and `seacas` doesn't test it in CI.